### DISCUSSION TITLE
How do I trigger an action using js?

### DISCUSSION THREAD

**Discussion by degodfather** [filamentphp/filament#12110] [Help] [answered]
### Package

Actions

### Package Version

v3.0.0

### How can we help you?

I want to open the following action modal using js rather than a button. 

`<div>
    {{ $this->deleteAction }}
 
    <x-filament-actions::modals />
</div>`

How's that possible?

**Comment by degodfather** [comment]
Hi,
I don't know whether this is the correct method or not, but I got it working with the following code.

`<script>   
@this.mountAction('delete');
</script>  `

**Reply by leandrocfe** [reply]
`@script` and `$wire`?
https://livewire.laravel.com/docs/javascript#using-wire-from-scripts

**Reply by degodfather** [reply, marked as answer]
@leandrocfe 
```@script
<script>
$wire.$call('functionname');
</script>
```
works! Thanks.
